.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
You shouldn't be adding the files in your `.idea` folder to the repo.  I'm not sure what these files are, and we don't want a bunch of extra files like this cluttering up the repo.